docs/getting-started-guides/juju.md

```diff
@@ -236,5 +257,11 @@ works with [Amazon Web Service](https://jujucharms.com/docs/stable/config-aws),
 [Vagrant](https://jujucharms.com/docs/stable/config-vagrant), and
 [Vmware vSphere](https://jujucharms.com/docs/stable/config-vmware).
 
-If you do not see your favorite cloud provider listed many clouds can be
-configured for [manual provisioning](https://jujucharms.com/docs/stable/config-manual).
+If you do not see your favorite cloud provider listed many clouds with ssh
+access can be configured for
+[manual provisioning](https://jujucharms.com/docs/stable/config-manual).
+
+The Juju Kubernetes work is curated by a very small group of community members.
+Let us know how we are doing. If you find any problems please open an
+[issue at the kubernetes project](https://github.com/kubernetes/kubernetes/issues)
+and tag it with "juju" so we can find it. 
```

Comment:
perfect! :+1: :+1: :+1: :+1: :+1:   I give this single line a full five thumbs up and want to merge it based on this line alone!